tests/components/zha/test_sensor.py

```diff
@@ -779,26 +779,26 @@ async def test_unsupported_attributes_sensor(
         (
             1,
             1232000,
-            "123.20",
+            "123.2",
             UnitOfVolume.CUBIC_METERS,
         ),
         (
             3,
             2340,
-            "0.23",
-            f"100 {UnitOfVolume.CUBIC_FEET}",
+            "0.65",
```

Comment:
why are the values and units changing for these?

Reply:
As we are setting a device class now, Home Assistant is converting to the users selected units.